include/stats/device.h

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2021 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
```

Comment:
wondering if wouldn't be better renaming this file to `i2c.h`. Otherwise it will become very unmaintainable when starting to add stats support for different kind of devices.

Reply:
That's a great point and something I was considering myself, it is relatively small but right, as time goes on... it could become large and hard to read, probably worth splitting now.